cloudevents/SDK.md

```diff
@@ -173,6 +173,7 @@ Undo it when done:
 | [CloudEvents Core](https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/spec.md)                                                       | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark:  | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark:    | :heavy_check_mark:  | :heavy_check_mark:  |
 | Event Formats |
 | [Avro](https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/formats/avro-format.md)                                                    | :heavy_check_mark: |     | :x:  | :x: |     |     |        |      | :x:  |
+| [Avro Turbo](https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/formats/avro-turbo-format.md)                                                    | :x: |     | :x:  | :x: |     |     |        |      | :x:  |
```

Comment:
same here - we may need to point to the HEAD version for now - so a relative link would be ok for the PR.